ci-operator/step-registry/openshift/upgrade/qe/test/custom-cli/openshift-upgrade-qe-test-custom-cli-chain.yaml

```diff
@@ -0,0 +1,17 @@
+chain:
+  as: openshift-upgrade-qe-test-custom-cli
+  steps:
+  - ref: idp-htpasswd
+  - ref: cucushift-pre
+  - ref: openshift-extended-upgrade-pre-custom-cli
+  - ref: cucushift-upgrade-prepare
+  - ref: cucushift-upgrade-prehealthcheck
+  - ref: cucushift-upgrade-toimage
+  - ref: cucushift-upgrade-healthcheck
```

Comment:
I see there is profile with none basecap in chain upgrade jobs, shall we keep step `cucushift-upgrade-check-capability` in the chain?

Reply:
Okay, later I will add it back.

Reply:
Done.